_posts/2021-04-14-wtd-writing-day.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Trino at Writing Day"
+author: Rose Williams (she/her)
+---
+
+First time Trino blogger, long time lurker on the Trino slack. My name is Rose
+Williams and I’m an open source docs enthusiast! I’ve had the pleasure of
+contributing to this community for the past few months. Recently I’ve been
+working with Brian Olsen, our fearless developer advocate, as well as some of
+our other Trino doc contributors, to get Trino ready for the Write the Docs
+[Writing Day](https://www.writethedocs.org/conf/portland/2021/writing-day/) open source event!
+
+If you’re not familiar with [Write the Docs](https://www.writethedocs.org), it’s
+a global community of people who care about documentation. 
+
+> “We consider everyone who cares about communication, documentation, and their
+> users to be a member of our community. This can be programmers, tech writers,
+> developer advocates, customer support, marketers, and anyone else who wants
+> people to have great experiences with software.”
+
+[Writing Day](https://www.writethedocs.org/conf/portland/2021/writing-day/) is
+the first day of their upcoming virtual documentation conference, [Write the
+Docs Portland (PST)](https://www.writethedocs.org/conf/portland/2021/) April
+25-27, 2021. The goal of Writing Day is to get a bunch of interesting people in
+a room together and introduce them to cool open source projects that they can
+onboard and contribute to.
+
+Writing Day is open to all conference attendees and several Trino members are
+attending as mentors. Leading up to the conference, we're focused on identifying
+docs issues that are ideal for first time contributors. If you’re a regular
+Trino contributor, you might notice that we’re going through and tagging items
+as “good first issue” and “docs” - we’ll be using those tags to create an issues
```

Comment:
Maybe link this https://github.com/trinodb/trino/issues?q=is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22